src/iuse_actor.cpp

```diff
@@ -2216,6 +2216,7 @@ void musical_instrument_actor::load( const JsonObject &obj )
     volume = obj.get_int( "volume" );
     fun = obj.get_int( "fun" );
     fun_bonus = obj.get_int( "fun_bonus", 0 );
+    description_frequency = time_duration::from_turns( obj.get_int( "description_frequency", 0 ) );
```

Comment:
I think the following line actually tries to read value from json and assign it to `description_frequency` variable.

Reply:
Yes, that's what I tried to do...Seems that I failed to make it clear in "describe the solution"....